adapters/nextmillennium/nextmillennium.go

```diff
@@ -127,7 +111,11 @@ func createBidRequest(prebidBidRequest *openrtb2.BidRequest, params *openrtb_ext
 		placementID = fmt.Sprintf("g%s;%s;%s", params.GroupID, size, domain)
 	}
 
-	bidRequest.Ext.Prebid.StoredRequest.ID = placementID
+	jsonExt := json.RawMessage(`{"prebid":{"storedrequest":{"id":"` + placementID + `"}}}`)
```

Comment:
There are security concerns with directly creating JSON this way with string content. Please create a data structure (anonymous structs are a good option) and use json.Marshal here.

Reply:
Thanks!
I changed it